BootloaderCommonPkg/Library/BootloaderDebugLib/DebugLib.c

```diff
@@ -75,24 +74,20 @@ DebugPrint (
   //
   // Send the print string to debug output handler
   //
-  if (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_LOG_BUFFER) {
+  if ((PcdGet32 (PcdDebugInterfaceFlags) & DEBUG_DEVICE_RAM) != 0) {
     DebugLogBufferWrite  ((UINT8 *)Buffer, Length);
   }
 
-  OutputToSerial = (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_SERIAL_PORT) ? TRUE : FALSE;
-  if (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_CONSOLE) {
-    ConsoleWrite ((UINT8 *)Buffer, Length);
+  ConsoleWrite ((UINT8 *)Buffer, Length);
```

Comment:
Why do we always do console write ?

Reply:
In general the new code has same logic.   ConsoleWrite will check if need output to console device.

Reply:
Debug output should not call ConsoleWrite() by default.  It needs to be controlled through PCD.   ConsoleLib itself is relative big.  We should not by default call this console interface in DebugLib in early stages when the size is limited.   Also user might not want to output all debug message into console device such as graphics console.